bitnami/elasticsearch/templates/_helpers.tpl

```diff
@@ -56,14 +56,20 @@ Return the hostname of every ElasticSearch seed node
 {{- define "elasticsearch.hosts" -}}
 {{- $clusterDomain := .Values.clusterDomain }}
 {{- $releaseNamespace := .Release.Namespace }}
+{{- if gt (.Values.master.replicas | int) 0 }}
 {{- $masterFullname := include "elasticsearch.master.fullname" . }}
-{{- $coordinatingFullname := include "elasticsearch.coordinating.fullname" . }}
-{{- $dataFullname := include "elasticsearch.data.fullname" . }}
-{{- $ingestFullname := include "elasticsearch.ingest.fullname" . }}
 {{- $masterFullname }}.{{ $releaseNamespace }}.svc.{{ $clusterDomain }},
+{{- end -}}
```

Comment:
Does the conditional (`replicas > 0`) fix any particular issue for the case of master? Since, if the amount of replicas is zero, shouldn't all other node types fail as well?

Reply:
It doesn't fix any known issue. It was just for the consistency when evaluating the hosts.

But I totally agree that if there is no master, the cluster will never boot properly.

Reply:
Oh, OK! I think it is good nevertheless.